www/secmail/views/forms/icla.js.rb

```diff
@@ -89,6 +89,7 @@ def componentDidMount()
     end
 
     @realname = name
+    @pubname = name
```

Comment:
Case 3 is the scenario I was talking about.  A secretary sends in a bunch of forms for a department of a large corporation.  Before this change (and in the secretary workbench) you would replace the name and then tab to the next field.  The onFocus handler would kick in and do the ||= assignment, which would fill in the field if currently empty, but leave it alone otherwise.